### ISSUE TITLE
Optimizer should display the initial force before starting iterations

### ISSUE THREAD

**Issue by GPMueller** [spirit-code/spirit#36]
Before, the initial force that was displayed was sometimes nonsense.
Currently, it is not displayed at all.

**Comment by GPMueller** [comment]
Implemented in e55ff5d931b36a1a3f89150f0b60fda45ff8040a

**Comment by GPMueller** [comment]
Merged into develop in 81b9a7f366c8b26cd3bf8a9b287507e87838878c
